### PR TITLE
Ethereal revival crystal reflects lasers now

### PR DESCRIPTION
**Pull request by GuillaumePrata**
## About The Pull Request
Just that, like titanium walls as an example.
## Why It's Good For The Game
A few days ago I watched an officer murder an traitor ethereal on the shuttle, then he stood there waiting for the crystal to pop and started blasting it with a laser gun.
The crystal broke, an extra laser bolt flew into the titanium wall behind the crystal and right back at the officer's face, putting them into crit.

This adds a minor gag to the crystals, it might murder someone a round, save someone in another, either way it is a funny interaction just waiting to happen.
## Changelog
:cl: Guillaume Prata
balance: Careful while shooting ethereal crystals with laser guns now. Another station officers' suffered serious burns when a laser bolt reflected on a crystal and went right back into their face.
/:cl:

### CODE REVIEW

File: code/modules/surgery/organs/heart.dm [hunk 1]
```diff
@@ -427,6 +427,7 @@
 	resistance_flags = FIRE_PROOF
 	density = TRUE
 	anchored = TRUE
+	flags_ricochet = RICOCHET_SHINY | RICOCHET_HARD
```
Comment (Fikou) [inline code/modules/surgery/organs/heart.dm:430]: RICOCHET_HARD means it reflects bullets

Comment (GuillaumePrata) [reply code/modules/surgery/organs/heart.dm:430]: The base structure has RICOCHET_HARD too so it is just keeping old behavior.

**Comment by Fikou** [comment]
oh right update your pr to 515 bozo

**Comment by GuillaumePrata** [comment]
A yes, I fucked up on the git fu again, I will reopen this later.

**Pull request closed without merging**